mail/views.py

```diff
@@ -73,6 +74,26 @@ def post(self, request, *args, **kargs):  # pylint: disable=unused-argument
         )
 
 
+class AutomaticEmailView(GenericAPIView):
```

Comment:
Ok, yeah I'm going to come back in a follow-up PR and add the method to handle the `PATCH` request (going to do some UI work first) so I'll keep it as-is.